src/Spec/Swagger2.php

```diff
@@ -312,6 +312,7 @@ public function getDefinitions()
                 "name" => $key,
                 "properties" => $schema['properties'] ?? [],
                 "description" => $schema['description'] ?? [],
+                "error_types" => $schema['x-appwrite']['types'] ?? [],
```

Comment:
There's no need to add this here. This will make the `isset($sch['error_types'])` always true down below